src/release_windows.go

```diff
@@ -37,7 +37,18 @@ func writeDesktopFiles(mc *metaConfig) {
 }
 
 func configureAutologin(mc *metaConfig) {
-	// set up user autologin
+	if verboseEnabled {
+		infoPrint("Setting Up autologin for " + mc.Config.User + "...")
+	}
+	powershellAutoLogin := `
```

Comment:
Pretty sure this still doesn't work if the key exists-- not 100% on that though.

Reply:
Yeah waiting on tanay to check that

Reply:
Powershell script tested, working